qe_metrics/utils/product_utils.py

```diff
@@ -42,8 +42,9 @@ def append_last_updated_arg(query: str, look_back_days: int) -> str:
         str: A query with an argument to filter issues that have been updated in the last 'look_back_days' days
     """
     if any(word in query for word in ["updatedDate", "updated"]):
-        raise ValueError(
+        LOGGER.error(
             f'Query is already using the "updatedDate" or "updated" field. Query will not be executed. \nQuery: "{query}"'
         )
+        return None
```

Comment:
can you return here empty string? it's always better to return the same type always.

Reply:
Should have thought of that, thanks!